README.md

```diff
@@ -72,11 +72,13 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
 | `PHONE_CARRIER` | [Supported carriers](#supported-carriers) for SMS; optional, email configuration required |
 | `RATE_LIMIT_TIMEOUT` | Rate limit timeout for each full store cycle; optional, default: `5000` |
 | `SLACK_CHANNEL` | Slack channel for posting (e.g., `update`); optional |
-| `SLACK_TOKEN` | Slack API token; optional
+| `SLACK_TOKEN` | Slack API token; optional |
 | `STORES` | [Supported stores](#supported-stores) you want to be scraped; optional, default: `nvidia` |
 | `OPEN_BROWSER` | Toggle for whether or not the browser should open when item is found, default: `true` |
 | `PLAY_SOUND` | Play this sound notification if a card is found.; optional |
 | `SCREENSHOT` | Capture screenshot of page on successful hit; optional, default `true` |
+| `TELEGRAM_BOT_TOKEN` | Telegram bot token; optional |
```

Comment:
What do you think about changing this to `TELEGRAM_TOKEN` or `TELEGRAM_ACCESS_TOKEN` just to be more related to Telegram?

Reply:
It always used to be called bot token. That terminology change must've been fairly recent, so let me fix the env var, sure.

Reply:
Oh, I actually don't know the terminology at all. I just saw the property name of the function you were calling was `accessToken`, so I figured it was the same.

Whatever you think makes sense then.